event-input.h

```diff
@@ -162,6 +162,46 @@ static const int *const touch_event_keys[]= {
 	touch_keys,
 };
 
+static const int keyboard_event_types[] = {
+	EV_KEY,
+	/** No more entries */
+	-1
+};
+
+static const int keyboard_keys[]= {
+	KEY_Q,
+	KEY_W,
+	KEY_E,
+	KEY_R,
+	KEY_T,
+	KEY_Y,
+	KEY_U,
+	KEY_I,
+	KEY_O,
+	KEY_P,
+	KEY_A,
+	KEY_S,
+	KEY_D,
+	KEY_F,
+	KEY_G,
+	KEY_H,
+	KEY_J,
+	KEY_K,
+	KEY_L,
+	KEY_Z,
+	KEY_X,
+	KEY_C,
+	KEY_V,
+	KEY_B,
+	KEY_N,
+	KEY_M,
+	-1
```

Comment:
Did you get that list from some documentation, that states like "if you have those keys, then it is a keyboard" or you made it up? Also, this will not match power button on n900, for example, so once we remove match by name (and we'll have to soon or later), we'll have lots of things broken. Please rethink that criteria.

Reply:
This is quite similar to how the kernel defines a keyboard, it defines  a kyboard as anything reporting keycodes <255. 
duplicating this behaviour would be tedious with the current code so i setteled on this.

this should explicitly NOT match devices with power keys that is handled by the next match down: power_event_types